CONTRIBUTING.md

```diff
@@ -3,18 +3,6 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
-## Contributor License Agreement
-
-Contributions to this project must be accompanied by a Contributor License
-Agreement. You (or your employer) retain the copyright to your contribution,
-this simply gives us permission to use and redistribute your contributions as
-part of the project. Head over to <https://cla.developers.google.com/> to see
-your current agreements on file or to sign a new one.
-
-You generally only need to submit a CLA once, so if you've already submitted one
-(even if it was for a different project), you probably don't need to do it
-again.
-
 ## Code reviews
 
 All submissions, including submissions by project members, require review. We
```

Comment:
So this is written but not accurate, because it's copied from xi-editor. I think now might not be a bad time to consciously design some process. I want to be able to push without review, partly because there isn't (yet) a critical mass of reviewers, partly because sometimes I need to push xi-win-ui changes to be able to use them in my other main project (synthesizer-io). But for other contributions, the process in xi-editor and xi-mac seems to be working pretty well, and I think we could adapt that.

In the longer term, we'll _definitely_ want to refactor xi-win-ui into a separate repo. Maybe we don't want to do that right now because there's a lot of work that requires changes to both the ui and the app, so that would increase friction.

Reply:
> I want to be able to push without review

I'm not aware of any major contributors to this repository besides yourself, so maybe changing the line to just `all PR's require review by Raph` would suffice, until there's that critical mass is attained?

> now might not be a bad time to consciously design some process

Besides the review process, are there any other thoughts you have on the xi-win repo that might be different from xi-editor/xi-mac?

> we'll _definitely_ want to refactor xi-win-ui into a separate repo

I _did_ wonder about that. Maybe the README should be updated to be a bit more clear, that there's a windows UI toolkit + the xi-win editor?